### PR TITLE
Add receptorctl ping doc

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

**Review by matoval — APPROVED** [review]

File: docs/source/receptorctl/receptorctl_ping.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <<pid>``
+
+``count`` (positive integer) is the number of pings to send (default is 4).
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_ping.rst]:
```suggestion
``count`` specifies the number of pings to send. The value must be a positive integer. The default is ``4``.
```

File: docs/source/receptorctl/receptorctl_ping.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+``delay`` (positive float) is the time (in seconds) to wait between pings (default = 1.0)
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_ping.rst]:
```suggestion
``delay`` specifies the time, in seconds, to wait between pings. The value must be a positive float. The default is ``1.0``.
```

File: docs/source/receptorctl/receptorctl_ping.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
+----
+ping
+----
+
+.. contents::
+   :local:
+
+``receptorctl ping`` Pings a receptornode.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_ping.rst]:
```suggestion
``receptorctl ping`` tests the network reachability of Receptor nodes.
```

Comment (oraNod) [reply docs/source/receptorctl/receptorctl_ping.rst]: Not sure if s/reachability/connectivity is better here or not. I guess there is some nuance there. "reachability" implies that you can verify the target node is up and active whereas "connectivity" implies the target node is available to receive inbound requests.

**Review by oraNod — APPROVED** [review]
LGTM. Thanks @thom-at-redhat

**Pull request merged**